ansible/playbooks/utils/mount_volume.yml

```diff
@@ -4,21 +4,18 @@
   hosts: all
   vars:
     example:
-      - "-e \"{'name':'/vol1', 'src':'/dev/vdc', 'fstype': 'ext3'}\""
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
```

Comment:
I don't see where this playbook uses `VOLUME_MKFS_DEVICE`, but it does use `VOLUME_MOUNT_LOCATION` which isn't in your example.